### PR TITLE
Hungarian translation for Mastodon.

### PR DESCRIPTION
**Pull request by kodacs**
I translated the software to hungarian, I hope I didn't made any mistake.

### CODE REVIEW

File: config/locales/devise.hu.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+en:
```
Comment (DeeUnderscore) [inline config/locales/devise.hu.yml]: You probably want this, and other yaml files, to say `hu`.

**Comment by kodacs** [comment]
@DeeUnderscore Thanks! You're absolutely right! I fixed that.

btw: testing translations? really?

**Comment by Gargron** [comment]
1) Travis tests all pull requests
2) We have tests for missing keys in translations, so it fails if something was forgotten (it only works for server-side translations though)

**Comment by kodacs** [comment]
Thanks for merging! 

I'll look into the missing keys problem, also update the translation, as it's far from perfect.

**Pull request merged**